File: Assets/AltUnityTester/AltUnityDriver/Commands/AltUnityCommands/AltUnitySetServerLogging.cs
using Altom.AltUnityDriver.Logging;

namespace Altom.AltUnityDriver.Commands
{
    public class AltUnitySetServerLogging : AltBaseCommand
    {
        private readonly AltUnitySetServerLoggingParams cmdParams;

        public AltUnitySetServerLogging(IDriverCommunication commHandler, AltUnityLogger logger, AltUnityLogLevel logLevel) : base(commHandler)
        {
            this.cmdParams = new AltUnitySetServerLoggingParams(logger, logLevel);
        }
        public void Execute()
        {
            this.CommHandler.Send(this.cmdParams);
            var data = this.CommHandler.Recvall<string>(this.cmdParams);
            ValidateResponse("Ok", data);
        }
    }
}

File: Bindings~/java/src/main/java/ro/altom/altunitytester/Commands/AltUnityCommands/AltUnitySetServerLogging.java
package ro.altom.altunitytester.Commands.AltUnityCommands;

import ro.altom.altunitytester.IMessageHandler;
import ro.altom.altunitytester.Commands.AltBaseCommand;

public class AltUnitySetServerLogging extends AltBaseCommand {

    private AltSetServerLoggingParams setServerLoggingParameters;

    public AltUnitySetServerLogging(IMessageHandler messageHandler,
            AltSetServerLoggingParams setServerLoggingParameters) {
        super(messageHandler);

        this.setServerLoggingParameters = setServerLoggingParameters;
    }

    public void Execute() {
        SendCommand(setServerLoggingParameters);
        recvall(setServerLoggingParameters, String.class);

    }

}
